src/test/java/net/oauth/jsontoken/AlwaysFailAudience.java

```diff
@@ -0,0 +1,16 @@
+package net.oauth.jsontoken;
+
+import com.google.gson.JsonObject;
+import java.security.SignatureException;
+
+/**
+ * Fails on any audience (even null).
+ */
+public class AlwaysFailAudience implements Checker {
```

Comment:
nit: final

Reply:
AlwaysFailChecker?

Reply:
I can make the `final` change in a new PR. I named it `AlwaysFailAudience` to be consistent with the existing `IgnoreAudience` class, since they are opposites. Would it be worth renaming both?

Reply:
Yes we can rename both. It doesn't look like they have anything to do with audience.

Reply:
I think the original intention of the checker class (from the documentation in the constructor in `JsonTokenParser`) was to validate audience. I do agree that the names of these two classes can be more descriptive though.

Reply:
It does seem to be the original intention but the interface itself is general and can be used for any checks.